OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitWebViewFragment.java

```diff
@@ -112,6 +116,7 @@ public void onPageFinished() {
                 if (authWebView.isPageLoaded()) {
                     fetchDateBannerInfo();
                     evaluateJavascriptForiFrame();
+                    evaluateXBlocksForBanner();
```

Comment:
As per discussion please change the priority for banners.
cc: @farhan-arshad-dev